python/paddle/utils/cpp_extension/extension_utils.py

```diff
@@ -343,8 +396,9 @@ def find_paddle_libraries(use_cuda=False):
     # pythonXX/site-packages/paddle/libs
     paddle_lib_dirs = [get_lib()]
     if use_cuda:
-        cuda_dirs = find_cuda_includes()
-        paddle_lib_dirs.extend(cuda_dirs)
+        cuda_lib_dirs = find_cuda_libraries()
+        print(cuda_lib_dirs)
```

Comment:
You can use `log_v` to log necessary log for users.

Reply:
Removed